cooluri/link.Functions.php

```diff
@@ -259,7 +259,7 @@ public static function pageNotFound($lConf) {
       if (!empty($lConf->cache->pagenotfound)) {
           switch ((string)$lConf->cache->pagenotfound->behavior['type']) {
               case 'message': $res = (string)$lConf->cache->pagenotfound->behavior; break;
-              case 'page':    $res = file_get_contents((string)$lConf->cache->pagenotfound->behavior); break;
+              case 'page':    $res = t3lib_div::getUrl((string)$lConf->cache->pagenotfound->behavior); break;
```

Comment:
Looking at it, the whole code could use some refactoring. The wrapper classes need interface and then be injected into the main class. So, I can accept this at the moment, but make it 6.2+ compatible. Just use \TYPO3\CMS\Core\Utility\GeneralUtility instead on t3lib_div and maybe, to be sure, add at beginning before "class"

if (!class_exists('\TYPO3\CMS\Core\Utility\GeneralUtility')) {
    class_alias('t3lib_div','\TYPO3\CMS\Core\Utility\GeneralUtility');
}